app/javascript/packages/normalize-yaml/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### New Features
 
 - Added new punctuation formatter to collapse multiple spaces to a single space.
+- Added `--double-quote` command line arg that overrides Prettier's `singleQuote` option with `false`
```

Comment:
I wonder if this could be an option that allows specifying a particular configuration file, so we could use a different configuration file altogether, rather than having one-off options for individual Prettier configuration values.

Reply:
Also, out of curiosity, _why_ do we want double-quotes?

Reply:
it just matched closer to the `.to_json` that I generated in Ruby. 🤷 I guess we could change to use double quotes?

Reply:
Reverted quoting changes in 1e44c5927692bc17346b52f0ad99c13c0cb38fed